src/core/src/op/topk.cpp

```diff
@@ -91,12 +91,18 @@ bool evaluate_topk(const HostTensorPtr& arg,
                    const element::Type index_et) {
     bool rc = true;
     switch (arg->get_element_type()) {
+        NGRAPH_TYPE_CASE(evaluate_topk, i8, arg, out_indices, out_values, out_shape, axis, k, max, sort, index_et);
```

Comment:
Why did you add new data types to topK? we should extend evaluate only in case of customer models, because it increases the binary size.

Reply:
I got it as a bug, and as I got it as ordinary, I'll close this PR.